lib/explorer/data_frame.ex

```diff
@@ -188,6 +188,17 @@ defmodule Explorer.DataFrame do
       iex> df["class"][3]
       1
 
+
+  When trying to access a column in the dataframe that does not exist, an `ArgumentError` is raised.
+
+      iex> df = Explorer.Datasets.wine()
+      iex> try do
+      ...>   {:ok, df["abcd"]}
+      ...> rescue
+      ...>   _ in ArgumentError -> :error
```

Comment:
```suggestion
      ...>   ArgumentError -> :error
```

Reply:
Good point, I think we can even be better than the try/catch. I dropped another suggestion :)

Reply:
That's way better :P